pkg/tfvars/baremetal/baremetal.go

```diff
@@ -114,16 +114,18 @@ func TFVars(numControlPlaneReplicas int64, libvirtURI, apiVIP, imageCacheIP, boo
 
 		// Host Details
 		hostMap := map[string]interface{}{
-			"name":                 host.Name,
-			"port_address":         host.BootMACAddress,
-			"driver":               accessDetails.Driver(),
-			"boot_interface":       accessDetails.BootInterface(),
-			"management_interface": accessDetails.ManagementInterface(),
-			"power_interface":      accessDetails.PowerInterface(),
-			"raid_interface":       accessDetails.RAIDInterface(),
-			"vendor_interface":     accessDetails.VendorInterface(),
-			"raid_config":          string(raidConfig),
-			"bios_settings":        string(biosSettings),
+			"name":                                 host.Name,
+			"port_address":                         host.BootMACAddress,
+			"bmc_address":                          host.BMC.Address,
+			"bmc_disable_certificate_verification": host.BMC.DisableCertificateVerification,
```

Comment:
The bmc address and flag to disable certificate validation already exists in the driver_info - see here:

https://github.com/openshift/installer/blob/master/pkg/tfvars/baremetal/baremetal.go#L78..L86

So I don't think we should add these new terraform APIs, the provider should just consume the existing `driver_info` data?

Reply:
It may be that `DisableCertificateVerification` can be derived from driver_info, but `Address` cannot. Please see the [detailed statement](https://github.com/openshift-metal3/terraform-provider-ironic/pull/53#discussion_r770262028) in PR53's comments.